### PR TITLE
Removes 'that' one (two actually) green NRI armor set(s)

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Gets rid of that 6b[whatever] green armor vest and helmet, as well as Altyn (ig) and green SWAT armor heavy armor, sprites and whatnot included.
Additionally gets rid of the NRI ERT's heavy role as it no longer has the armor and was frankly just detrimental.

## How This Contributes To The Skyrat Roleplay Experience
It sucks; "new" (it's been a couple months) surplus armor does the job better.
NRI ERT heavy is unmaintaned, nor would be, and as such I wanted to get rid of them for a while; because the standard role does their job better.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  It compiles.
</details>

## Changelog

:cl: Stalkeros
del: Removes green NRI armor vest and helmet, heavy variants included.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
RIP that really cool helmet

**Review by GoldenAlpharex — APPROVED** [review]
Sure.

File: modular_skyrat/modules/novaya_ert/code/suit.dm [hunk 1]
```diff
@@ -8,84 +8,6 @@
...
-	bomb = 35
-	fire = 50
-	acid = 50
-	wound = 15
-
-/obj/item/clothing/suit/armor/heavy/nri
-	name = "\improper Cordun-M armor system"
-	desc = "A robust set of full-body armor designed for the harshest of environments. A modern set of heavy armor recently implemented by NRI Defense Collegium to accomodate with modern specifications. While a combination of lighter materials and a passive internal exoskeleton might assist the user's movement, you'll still be as slow as a snail."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits/armor.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suits/armor.dmi'
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/suit.dm:47]: Actually I think you forgot to remove the worn icons from their dmi for the suits here. You only did the Teshari ones.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/novaya_ert/code/suit.dm (hunk 1 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/novaya_ert/code/suit.dm:47]: > Actually I think you forgot to remove the worn icons from their dmi for the suits here. You only did the Teshari ones.

i forgor :skull:

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**